src/utils/index.js

```diff
@@ -209,3 +212,23 @@ export function match(url, pattern) {
   }
   return minimatch(url, pattern);
 }
+
+export function readFile(fileName) {
```

Comment:
Better rename it to `readYaml` and handle yaml errors here too vs extra try-catch in every place.

Also, make sure to supply filename to yaml.safeLoad so it prints a nicer error.

Reply:
done